### PR TITLE
Add option to disable sd outbound report

### PR DESCRIPTION
**Pull request by bianpengyuan**

### CODE REVIEW

**Review by douglas-reid — APPROVED** [review]

**Comment by howardjohn** [comment]
Can you link to an issue or add a description of the motivation behind this?

**Comment by bianpengyuan** [comment]
@howardjohn sure, just created one https://github.com/istio/istio/issues/22333

File: operator/pkg/apis/istio/v1alpha1/values_types.proto [hunk 1]
```diff
@@ -1354,6 +1354,8 @@ message TelemetryV2StackDriverConfig {
 
   google.protobuf.BoolValue topology = 4;
 
+  google.protobuf.BoolValue disableOutbound = 6;
```
Comment (howardjohn) [inline operator/pkg/apis/istio/v1alpha1/values_types.proto]: are we just going to have some request a `disableInbound` flag a couple of weeks from now, or do we think this is a good long term API?

Comment (bianpengyuan) [reply operator/pkg/apis/istio/v1alpha1/values_types.proto]: Disable client side reporting should be a general enough option that will last for long term.

**Review by richardwxn — APPROVED** [review]

**Pull request merged**